packages/atomic-angular/package.json

```diff
@@ -39,8 +39,8 @@
     "karma-jasmine": "5.1.0",
     "karma-jasmine-html-reporter": "2.1.0",
     "ncp": "2.0.0",
-    "ng-packagr": "17.3.0",
-    "typescript": "5.4.5"
+    "ng-packagr": "18.2.1",
+    "typescript": "5.5.4"
```

Comment:
Uh, why the typescript downgrade? Every other package is at 5.4.5

Reply:
and I guess you meant upgrade ?

Reply:
I decided to leave it as is. We will upgrade typescript soon anyway